app/src/main/res/layout/all_episodes_fragment.xml

```diff
@@ -17,6 +17,41 @@
         tools:itemCount="13"
         tools:listitem="@layout/new_episodes_listitem" />
 
+    <LinearLayout
+        android:id="@+id/llEmpty"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:gravity="center">
+
+        <TextView
+            android:id="@+id/emptyHeader"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:gravity="center"
+            android:text="@string/no_all_episodes_head_label"
+            android:paddingBottom="15dp"
+            android:paddingLeft="10dp"
+            android:paddingRight="10dp"
+            android:textSize="20dp"
```

Comment:
Text sizes should always be specified in `sp`, not `dp`, because `sp` respects the font size that is set in Android's settings menu. Same in the other xml files